docs/core/install/localized-intellisense.md

```diff
@@ -1,47 +1,47 @@
 ---
 title: Install localized IntelliSense files
-description: Learn how to set up your development machine to use localized IntelliSense files for .NET Core projects in Visual Studio.
-ms.date: 01/23/2020
+description: Learn how to set up your development machine to use localized IntelliSense files for .NET 5+ projects (including .NET Core) in Visual Studio.
+ms.date: 11/06/2020
 ---
-# How to install localized IntelliSense files for .NET Core
+# How to install localized IntelliSense files for .NET
 
-[IntelliSense](/visualstudio/ide/using-intellisense) is a code-completion aid that is available in different integrated development environments (IDEs), such as Visual Studio. By default, when you're developing .NET Core projects, the SDK only includes the English version of the IntelliSense files. This article explains:
+[IntelliSense](/visualstudio/ide/using-intellisense) is a code-completion aid that's available in different integrated development environments (IDEs), such as Visual Studio. By default, when you're developing .NET projects, the SDK only includes the English version of the IntelliSense files. This article explains:
 
 - How to install the localized version of those files.
 - How to modify the Visual Studio installation to use a different language.
 
 ## Prerequisites
 
-- [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core) or a later version.
+- [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core) or a later version, such as [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0).
```

Comment:
If you can use a later version, why not use that as the default? Also, I wonder if the visual studio can be upgraded to latest?

Reply:
I think prerequisites are usually the minimum required version.